src/search/components/TotalCount/TotalCount.tsx

```diff
@@ -0,0 +1,17 @@
+import { useSearchParams } from 'react-router-dom';
+
+import { useGetSearch } from '@/search/hooks/useGetSearch';
+import { StyledTotalCount } from '@/search/pages/Search/Search.style';
+
+export const TotalCount = () => {
+  const [searchParams] = useSearchParams();
+  const query = searchParams.get('query');
```

Comment:
```suggestion
  const query = searchParams.get('query') ?? '';
```

요기도 `searchParams.get()`을 호출하는 부분에서 기본값을 설정해주는게 좋을 것 같아요